example/lib/page_remote_config.dart

```diff
@@ -223,33 +254,74 @@ class _RemoteConfigPageState extends State<RemoteConfigPage> {
 
     /// Downloads all test variants
     /// Return back to [Contents]
-    void getAllTestVariants() {
-      Countly.instance.remoteConfig.testingGetAllVariants();
-      showCountlyToast(context, 'Downloaded All Variants', null);
+    Future<void> getAllTestVariants() async {
+      String resultString = '';
+      Map<String, List<String>> result = await Countly.instance.remoteConfig.testingGetAllVariants();
+      result.forEach((key, value) {
+        resultString += '\n[$key]:\n';
+        value.forEach((item) {
+          resultString += '- [$item]\n';
+        });
+      });
+      if (context.mounted) {
+        showCountlyToast(context, resultString, null);
+      }
+      print(resultString);
     }
 
     /// Downloads specific test variants
     /// Return back to [Contents]
-    void getSpecificTestVariants() {
-      Countly.instance.remoteConfig.testingGetVariantsForKey(rcKey);
-      showCountlyToast(context, 'Downloaded All Variants', null);
+    Future<void> getSpecificTestVariants() async {
+      String resultString = '';
+      List<String> result = await Countly.instance.remoteConfig.testingGetVariantsForKey(rcKey);
+      result.forEach((item) {
+        resultString += '- [$item]\n';
+      });
+      print(resultString);
+      if (context.mounted) {
+        showCountlyToast(context, resultString, null);
+      }
     }
 
 // Experiment Information -------------------------------
-    /// Downloads experiment information and prints it
+    /// Downloads experiment information
     /// Return back to [Contents]
     void downloadExperimentInfo() {
-      Countly.instance.remoteConfig.testingDownloadExperimentInformation((rResult, error) async {
-        if (rResult == RequestResult.success) {
-          Map<String, ExperimentInformation> experimentInfoMap = await Countly.instance.remoteConfig.testingGetAllExperimentInfo();
-          print(experimentInfoMap);
-          if (context.mounted) {
-            showCountlyToast(context, 'Experiment Info:${experimentInfoMap}', null);
-          }
+      String message = 'Downloaded experiment information';
+      Color? color = null;
+      Countly.instance.remoteConfig.testingDownloadExperimentInformation((rResult, error) {
+        if (error != null) {
```

Comment:
You should print `rResult` if error == null.

Reply:
im 50/50 on this